api/V1/Event.ts

```diff
@@ -194,4 +194,38 @@ export default function (app: Application, baseUrl: string) {
       });
     })
   );
+
+  /**
+   * @api {get} /api/v1/events/powerEvents Query power events for devices
+   * @apiName QueryPower
+   * @apiGroup Events
+   *
+   * @apiUse V1UserAuthorizationHeader
+   * @apiParam {Integer} [deviceId] Return only errors for this device id
+   *
+   * @apiSuccess {JSON} array of device power events
+   * @apiSuccessExample {json} Success-Response:
+   *     HTTP/1.1 200 OK
+   *     "events": [{
+   *       "lastReported": "2021-03-09 08:00:00",
+   *       "lastStarted": "2021-03-09 18:00:00",
+   *       "lastStopped": "2021-03-10 08:00:00",
+   *       "Device": {"id": 10, "devicename": "Camera2", "GroupId": 22, "Group":{"groupname": "examplegroup"}},
```

Comment:
Can we flatten the groupname?  So it's device.groupname not device.group.groupname.  

This is something that Jon has pointed out we should be doing - keeping our apis cleaner. Lets have a skype call with the whole team about this. 

Even better would be to get the group name from the request of admins for the group because otherwise we are just repeating stuff.